### PR TITLE
Adjusted IRS alignment time, ECAM memo, and effect on GPWS buttons

### LINKED ISSUE #500: [BUG]GWPS Terrain and Sys (overhead) should be in fault until Adirs / GPS aligned
**Mod Version**
Latest Master 0.2.0 version

**Describe the bug**
The GWPS Terrain and Sys buttons should show as faults until the global position of the plane is known (ie until NAV is aligned).
This is because the Aircraft doesnt know where it is until the ADirs and GPS are aligned. So It cant show terrain (hence these should be in fault).

**To Reproduce**
1. Cold and dark start   
2. batteries 1 and 2 on.
3. Ext power on.
4. You will see Terr and Sys buttons are not in fault.

**Expected behavior**
The Terr and Sys buttons should show Fault lights until the Adirs is aligned.

**Actual behavior**
Sys and Terrain buttons are not in fault whilst Adirs is not aligned.

**References**
refer to 5:00min in on the below a320 real world sim pilot cold and dark tutorial.
https://www.youtube.com/watch?v=Q0OKXZyLbBQ

**Additional context**

### PR DESCRIPTION
**Pull request by wpine215**
Fixes #500 

**Summary of Changes**

- IRS alignment times based on latitude have been re-calculated
- IRS alignment ECAM memo now skips "IR IN ALIGN 0 MN" to match the real aircraft
- IRS alignment ECAM memo now skips "IR IN ALIGN 7 MN" to match the real aircraft
- IRS alignment ECAM memo now rounds up when converting seconds to minutes
- GPWS Terrain button now displays FAULT when IRS not aligned
- GPWS Sys button now displays FAULT when IRS not aligned

- IRS ECAM memo now goes up to >10 MN instead of >7 MN
- "IR IN ALIGN" replaced with updated "IRS IN ALIGN"

**Screenshots (if necessary)**

image

**Additional context**

image

### CODE REVIEW

File: A32NX/ModelBehaviorDefs/Asobo/Airliner/Airbus.xml [hunk 1]
```diff
@@ -1190,6 +1190,24 @@
 	</UseTemplate>
 </Template>
 
+<Template Name="ASOBO_AIRLINER_GPWS_ALIGN_Template">
```
Comment (AdenFlorian) [inline A32NX/ModelBehaviorDefs/Asobo/Airliner/Airbus.xml:1193]: If this is a new custom template, we've started using A32NX_ in the name instead of ASOBO_

**Comment by AdenFlorian** [comment]
Did this end up making align take even longer?

**Comment by AdenFlorian** [comment]
Where are you gettings the "IRS IN ALIGN.." and the 10 minutes max from?

Contradicts these photos i found in the references channel:
image

image

**Comment by wpine215** [comment]
Reverted the align message changes for now. The screenshot is from an A320neo FCOM that becas22 (A320 pilot) referenced on the discord channel here: https://discordapp.com/channels/738864299392630914/739223513600360450/753222432231260223

**Comment by wpine215** [comment]
Also modified the template prefix, thanks for the heads up.

The IRS alignment time should be way more realistic now, with approximately 5 minute align times near the equator (for example, Bogota or Quito), 6-8 minute align times for the continental US, and ~11-12 minute align times for cities near 60-65 degrees latitude, such as Anchorage.

**Review by AdenFlorian — APPROVED** [review]
LGTM!

**Pull request merged**